source/localizable/contributing/repositories.md

```diff
@@ -1,7 +1,7 @@
-Ember is made up of several libraries. If you wish to add a feature or fix a bug please file a pull request against the appropriate repository. Be sure to check the libraries listed below before making changes in the Ember.js repository.
+Ember is made up of several libraries. If you wish to add a feature or fix a bug please file a pull request against the appropriate repository. Be sure to check the libraries listed below before making changes in the Ember repository.
 
 # Main Repositories
-**Ember.js** - The main repository for Ember.
```

Comment:
I think this is the one place where we would want to make an exception, since the repository is actually named ember.js. What do you think?